### PR TITLE
Added docs about how to use ServiceAccount auth

### PR DESCRIPTION
**Pull request by bkoski**

### CODE REVIEW

**Comment by gimite** [comment]
Thanks, it's a nice document. But I feel it makes README too long if it includes such detailed information. Can you add your doc to the document of GoogleDrive.login_with_oauth instead?:
https://github.com/gimite/google-drive-ruby/blob/master/lib/google_drive.rb#L10
It also contains detailed usage of OAuth too.

You may point to the method doc from README.

Also, use GoogleDrive.login_with_oauth instead of GoogleDrive::Session.new which is an undocumented method.

**Comment by bkoski** [comment]
Just updated with docs inline above the login_with_oauth, rather than in the README.

File: lib/google_drive.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Author: Hiroshi Ichikawa <http://gimite.net/>
 # The license of this source is "New BSD Licence"
 
+Note that when using the OAuth flow, the client will be able to access all files belonging to the the user who generated the token. To set permissions more specifically, you may want to consider using a service account.
```
Comment (gimite) [inline lib/google_drive.rb]: Putting this line here would cause syntax error in Ruby. Maybe you can add it just above the comment you added below?

**Comment by gimite** [comment]
Thanks! Merged.

**Pull request merged**